app/controllers/verify_controller.rb

```diff
@@ -2,9 +2,16 @@ class VerifyController < ApplicationController
   include RenderConditionConcern
   include IdvSession
 
+  STEP_NAMES = %w[
+    password_confirm
+    personal_key
+    personal_key_confirm
+  ].freeze
```

Comment:
Elsewhere I may have `.to_set`'d this, but based on my reading, there's no guarantee that a Set would preserve order, which we rely on here (see `first_step`).